dockerfiles/theia/Dockerfile

```diff
@@ -163,5 +163,6 @@ COPY --from=builder /home/theia-dev/theia-source-code/production/plugins /defaul
 COPY --chown=theia:root --from=builder /home/theia-dev/theia-source-code/production /home/theia
 USER theia
 WORKDIR /projects
+COPY src/get-webview-route.js ${HOME}/get-webview-route.js
```

Comment:
Should it be possible (maybe later) to use a runtime information instead of using environment variable.

Explanation: Here we're using a shell script before running the theia process in the container (by setting environment variable)

I think it would be cleaner if we bring a hook/extension point of theia at runtime by updating the setting on the fly. So no need to use separate script and fix env variable. It's when theia webview will resolve stuff, it will deal with che-theia code

Reply:
like to override this class: https://github.com/eclipse-theia/theia/blob/fc49b2c3b8a690ba5b94e1fd48cf17469b51057d/packages/plugin-ext/src/main/browser/webview/webview-environment.ts#L35

Reply:
Yes, I think we can do it in runtime.

Reply:
I think we should try, but as for now I would like to merge current PR as it is tested and fixes (for https case) blocker issue.

Reply:
yes this is why I said later